--- gxine-0.5.906/src/console_output.h.glib	2011-11-28 09:12:18.913892130 +0100
+++ gxine-0.5.906/src/console_output.h	2011-11-28 09:13:16.919895374 +0100
@@ -21,7 +21,7 @@
 #ifndef GXINE_CONSOLE_OUTPUT_H
 #define GXINE_CONSOLE_OUTPUT_H
 
-#include <glib/gmessages.h>
+#include <glib.h>
 
 void console_output_init (void);
 
--- gxine-0.5.906/src/engine.c.glib	2011-11-28 09:12:24.656892107 +0100
+++ gxine-0.5.906/src/engine.c	2011-11-28 09:13:11.047894891 +0100
@@ -30,7 +30,7 @@
 #include <errno.h>
 #include <unistd.h>
 
-#include <glib/gthread.h>
+#include <glib.h>
 
 #include "engine.h"
 #include "ui.h"
--- gxine-0.5.906/src/main.c.glib	2011-11-28 09:12:28.726891912 +0100
+++ gxine-0.5.906/src/main.c	2011-11-28 09:13:03.658894444 +0100
@@ -46,7 +46,6 @@
 #include <gdk/gdk.h>
 #include <gdk/gdkkeysyms.h>
 #include <glib.h>
-#include <glib/gmessages.h>
 
 #include "desktop_integration.h"
 #include "engine.h"
